frontend/src/scenes/surveys/SurveyAppearance.tsx

```diff
@@ -122,6 +128,24 @@ export function SurveyAppearance({
                             />
                         </>
                     )}
+                    <div className="mt-2">
+                        <LemonCheckbox
+                            label={
+                                <div className="flex items-center">
+                                    <span>Hide PostHog branding</span>
+                                    {!whitelabelAvailable ? (
+                                        <Tooltip title="Upgrade to PostHog Scale to hide PostHog branding">
```

Comment:
It'd be nice if we could do a CTA here that links them to the billing page.

```suggestion
                                        <Tooltip title="Upgrade to any paid plan to hide PostHog branding">
```

Reply:
I actually ended up removing that tooltip altogether there because it was flickering like crazy when used with the `disabledReason` tooltip